pkg/pillar/cmd/downloader/downloader.go

```diff
@@ -33,14 +33,16 @@ const (
 
 // Go doesn't like this as a constant
 var (
-	downloaderObjTypes = []string{types.AppImgObj, types.BaseOsObj,
-		types.CertObj}
+	debug              = false
+	debugOverride      bool                               // From command line arg
+	downloadGCTime     = time.Duration(600) * time.Second // Unless from GlobalConfig
+	downloadRetryTime  = time.Duration(600) * time.Second // Unless from GlobalConfig
+	downloaderObjTypes = []string{types.AppImgObj, types.BaseOsObj, types.CertObj}
+	Version            = "No version specified" // Set from Makefile
+	nilUUID            uuid.UUID
```

Comment:
This nilUUID  should really be a const, and if not be flagged as being a const. For some reason golang doesn't allow us to specify it has a const,

Reply:
every type has a "default" nil value. By doing `var nilUUID uuid.UUID` we are setting the `var` to the default nil value. To set a constant, we would need to set it to something. It is a bit strange that there is no sort of `if isNilValue(x)` or `if x == nilValue(uuid.UUID)`, but there you have it. 

Given that we can do nothing with it, I will just flag it with a comment.

Reply:
Updated.